packages/core/admin/admin/src/pages/MarketplacePage/PluginCard/index.js

```diff
@@ -1,185 +1,110 @@
-      short:
-        this.props.plugin.id === 'support-us' ? (
-          <FormattedMessage id={this.props.plugin.description.short} />
-        ) : (
-          this.props.plugin.description.short
-        ),
-      long:
-        this.props.plugin.id === 'support-us' ? (
-          <FormattedMessage
-            id={this.props.plugin.description.long || this.props.plugin.description.short}
-          />
-        ) : (
-          this.props.plugin.description.long || this.props.plugin.description.short
-        ),
-    };
-
-    return (
-      <Wrapper className={this.state.boostrapCol}>
-        <div className="wrapper">
-          <div className="cardTitle">
-            <div className="frame">
-              <span className="helper" />
-              <img src={this.props.plugin.logo} alt="icon" />
-            </div>
-            <div
-              onClick={e => {
-                // FIXME: dead link as we are changing the naming. Would be better to use a url comming from the api call directly
-                window.open(
-                  `https://github.com/strapi/strapi/tree/master/packages/strapi-plugin-${this.props.plugin.id}`,
-                  '_blank'
-                );
-              }}
-            >
-              {this.props.plugin.name} <i className="fa fa-external-link-alt" />
-            </div>
-          </div>
-          <div className="cardDescription">{descriptions.long}</div>
-          <div className="cardFooter" onClick={e => e.stopPropagation()}>
-            <div className="cardFooterButton">
-              <CheckPermissions permissions={adminPermissions.marketplace.install}>
-                <Button
-                  className={`${buttonClass} button`}
-                  label={buttonLabel}
-                  type="button"
-                  onClick={this.handleDownloadPlugin}
-                />
-              </CheckPermissions>
-            </div>
-            {this.props.isAlreadyInstalled ? (
-              settingsComponent
-            ) : (
-              <div className="compatible">
-                <i className={`fa fa-${this.props.plugin.isCompatible ? 'check' : 'times'}`} />
-                <FormattedMessage
-                  id={`app.components.PluginCard.compatible${
-                    this.props.plugin.id === 'support-us' ? 'Community' : ''
-                  }`}
-                />
-              </div>
-            )}
-          </div>
+import styled from 'styled-components';
+import { useIntl } from 'react-intl';
+import { Box } from '@strapi/design-system/Box';
+import { Stack } from '@strapi/design-system/Stack';
+import { Typography } from '@strapi/design-system/Typography';
+import { Button } from '@strapi/design-system/Button';
+import { LinkButton } from '@strapi/design-system/LinkButton';
+import { Flex } from '@strapi/design-system/Flex';
+import ExternalLink from '@strapi/icons/ExternalLink';
+import Duplicate from '@strapi/icons/Duplicate';
+
+const PluginLogo = styled.img`
```

Comment:
Not so sure about this component especially with the `width` and `height` properties, could you use as `<Box as="img" />` to apply the borderRadius property and use padding on the wrapper instead?